internal/reflect/outof_test.go

```diff
@@ -0,0 +1,112 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package reflect_test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/hashicorp/terraform-plugin-framework/attr"
+	"github.com/hashicorp/terraform-plugin-framework/diag"
+	refl "github.com/hashicorp/terraform-plugin-framework/internal/reflect"
+	"github.com/hashicorp/terraform-plugin-framework/path"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+)
+
+func TestFromValue_go_types(t *testing.T) {
+	t.Parallel()
+
+	// Used for testing nil -> null attr.Value
+	var nilSlice []string
```

Comment:
Pretty sure you can use `([]string)(nil)` or `new([]string)` directly for these 👍

Reply:
Ah! Thank you for this, I'm not sure why my brain couldn't come up with this lol